src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/partials/VoteStatuses/VoteStatuses.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import MembersAvatars from '~v5/shared/MembersAvatars';
+import { VoteStatusesProps } from './types';
+import { Watcher } from '~types';
+import { useMemberAvatars } from '../../hooks';
+import ProgressBar from '~v5/shared/ProgressBar';
+import Icon from '~shared/Icon';
+import { MotionVote } from '~utils/colonyMotions';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.Motion.steps.OutcomeStep.partials.VoteStatusesList';
+
+const VoteStatuses: FC<VoteStatusesProps> = ({ list }) => {
+  const { watchers } = useMemberAvatars();
+
+  return (
+    <>
+      {list.map(({ id, iconName, label, progress, status }) => (
+        <div key={id} className="flex items-center w-full gap-8">
+          <div className="flex flex-col gap-1 w-full">
```

Comment:
You shouldn't add `w-full` class here, if there's something next to this element, because it can break on some browsers. It's better to use flex grow and flex shrink properties in such cases. Flex 1 won't work as expected.